google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/compute/GoogleComputeEngineServiceAdapter.java

```diff
@@ -173,6 +181,18 @@
       // Add lookup for InstanceToNodeMetadata
       diskToSourceImage.put(instance.get().disks().get(0).source(), template.getImage().getUri());
 
+      boolean windows = options.autoCreateWindowsPassword() != null
+    		  ? options.autoCreateWindowsPassword()
+    	      : (OsFamily.WINDOWS == template.getImage().getOperatingSystem().getFamily());
```

Comment:
I think what @nacx means is that maybe you should simply remove this boolean windows and check at line 188 something like

```
if (options.autoCreateWindowsPassword() != null && OsFamily.WINDOWS == template.getImage().getOperatingSystem().getFamily() ) { ... }
```
thoughts?